PokeAlarm/Events/RaidEvent.py

```diff
@@ -176,6 +180,8 @@ def generate_dts(self, locale, timezone, units):
             'gym_name': self.gym_name,
             'gym_description': self.gym_description,
             'gym_image': self.gym_image,
+            'gym_sponsor': self.gym_sponsor,
```

Comment:
If we are going to leave this in, we should probably change to something like `is_sponsored` so it's clear it will be a True/False value.

Reply:
That won't be a true or false value. That will be the integer representing the sponsor. It is only the filter that would be true or false